sdk/dynamicconfig/metric/controller/remote/mock/notifier.go

```diff
@@ -15,29 +15,32 @@
 package mock
 
 import (
-	"go.opentelemetry.io/contrib/sdk/dynamicconfig/metric/controller/notify"
+	pb "github.com/open-telemetry/opentelemetry-proto/gen/go/experimental/metricconfigservice"
+	"go.opentelemetry.io/contrib/sdk/dynamicconfig/metric/controller/remote"
 )
 
-type Notifier struct {
-	data chan *notify.MetricConfig
+type Monitor struct {
```

Comment:
Should the file be renamed as monitor as well?

Reply:
Oops! Good catch, renaming